docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -129,6 +129,50 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
 
 :::
 
+### Link Cards
+
+Utilisez le composant `<LinkCard>` pour créer un lien vers différentes pages.
```

Comment:
In the English we say “link prominently” to distinguish this from just a regular link. Maybe something like this could accomplish the same here?

```suggestion
Utilisez le composant `<LinkCard>` pour créer un lien bien visible vers différentes pages.
```